static/app/views/onboarding/components/useOnboardingQueryParams.tsx

```diff
@@ -31,6 +36,7 @@ export function useOnboardingQueryParams(): [
       product: decodeList,
       showLoader: decodeBoolean,
       showManualSetup: decodeBoolean,
+      installationMode: decodeScalar,
```

Comment:
also it would be better to return `MANUAL` or `AUTO` from an enum but I did not want to add a custom `decode` here now